src/Events/Menus/Event.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Admin Home for TEC plugins.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Menus
+ */
+
+namespace TEC\Events\Menus;
+
+use TEC\Common\Menus\Abstract_Menu;
+use TEC\Common\Menus\Traits\CPT;
+use TEC\Common\Menus\Traits\Submenu;
+use Tribe__Events__Main;
+
+/**
+ * Class Events CPT admin/menu.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Menus
+ */
+class Event extends Abstract_Menu {
+	use Submenu, CPT;
```

Comment:
Following along the lines of my proposal in the Common PR: I think it would make more sense to have a base, `abstract`, custom-post-type, sub-menu class in the Common portion that would be used here.

Reply:
Haha! I asked those same questions in a call with Stephen yesterday :D He explained that he thinks the Traits make sense when combinations of the following are possible:

Menu + CPT + Admin Bar
Menu + CPT + Admin Bar + Settings
Menu + CPT
Submenu + CPT + Admin Bar
Submenu + Admin Bar
Submenu + Settings
etc.

A bunch of combinations are possible and we use a number of those combinations, so Traits feel like they make sense here.

Side note: I think the menu handling code could be spun out to its own StellarWP library!

Reply:
> Side note: I think the menu handling code could be spun out to its own StellarWP library!

That's my hope 🤞🏼